README.md

```diff
@@ -102,6 +102,7 @@ a double dash to prevent input files being used as option arguments:
                                 sequences.
     --config-file <file>        Read `minify()` options from JSON file.
     -d, --define <expr>[=value] Global definitions.
+    --ecma <version>            Specifiy supported level of ECMAScript.
```

Comment:
Here's the thing - most projects are using `uglify-es` as a drop-in replacement for `uglify-js@3`. We want to accept ES5 input and produce ES5 output by default. We still want to output ES5 properties, not shorthand properties by default. Ditto for compress - we don't want to convert ES5 anonymous functions to ES6 arrow functions by default.

It seems counter-intuitive but it works best to default to ES5 for compress/output by default, and higher ecma levels should be an explicit opt-in.